markdown_source/docs/about/roadmap.md

```diff
@@ -1,8 +1,36 @@
-<h1>Planned Features</h1>
+<h1>Roadmap Features</h1>
 After the v1.0.0 NREL-provided software distribution, the OpenStudio Application will continue to be supported with open source contributions by The OpenStudio Coalition and members of the software community. Plans include support for releasing new versions of the OpenStudio Application that stay aligned with scheduled releases of the OpenStudio SDK, with minimal or no changes to the user interface. Additionally, critical bugfix releases may be provided, on an as-needed basis. 
 
 The OpenStudio SDK, OpenStudio Command Line Interface (CLI), OpenStudio Server, OpenStudio Standards, OpenStudio Measures, Floorspace.JS Widget (Floorspace), Building Component Library (BCL), and Parametric Analysis Tool (PAT) will be unaffected. However, these tools will continue to be developed, distributed, and maintained by NREL. See [https://www.openstudio.net/new-future-for-openstudio-application](https://www.openstudio.net/new-future-for-openstudio-application) for details. 
 
 ## Partnerships and Open Source Software Contribution Opportunities
 
-Effective early May 2020, NREL will no longer distribute the OpenStudio Application and the SketchUp Plugin. The final NREL software distribution of the OpenStudio Application, along with any non-NREL future releases, will be available at [https://github.com/NREL/OpenStudioApplication/releases](https://github.com/NREL/OpenStudioApplication/releases). Companies, individual contributors, and organizations that are interested in providing development contributions, support and maintenance activities for the OpenStudio Application, including new features, defect fixes and enhancements, are encouraged to contact OSC@openstudiocoalition.org.
\ No newline at end of file
+Effective early May 2020, NREL will no longer distribute the OpenStudio Application and the SketchUp Plugin. The final NREL software distribution of the OpenStudio Application, along with any non-NREL future releases, will be available at [https://github.com/NREL/OpenStudioApplication/releases](https://github.com/NREL/OpenStudioApplication/releases). Companies, individual contributors, and organizations that are interested in providing development contributions, support and maintenance activities for the OpenStudio Application, including new features, defect fixes and enhancements, are encouraged to contact OSC@openstudiocoalition.org.
+
+## List of Features Needing Funding
+
+###Distributed Energy Resources
+   - Expand capabilities for the app to model solar, storage, electric vehicles, and heat pumps (water and space conditioning)
+
+###BCL Expansion
+   - Develop and provide guidance on building component library (BCL) measures to allow for decarbonization analysis
+   - Develop measures for REopt and/or SAM to allows energy efficiency measures to be modeled along with DERs
+   - Integrate U.S. DOE National Lab demand flexibility algorithms via BCL measures 
```

Comment:
```suggestion
   - Develop and support building component library (BCL) measures to allow for decarbonization analysis.  This could include measures which integrate REopt, SAM, demand flexibility, or other DER focused tools.
```

Reply:
@marccosta I tried to edit this to sound less DOE/NREL specific